app/views/users/show.html.erb

```diff
@@ -19,16 +21,54 @@
         </div>
         <br/>
         <h4><%= t('users.your_wallets') %></h4>
-        <div class="card mt-3">
+        <div class="card mt-3" style="border: none;">
           <ul class="list-group list-group-flush">
             <% @user.wallets.each do |wallet| %>
               <li class="list-group-item d-flex justify-content-between align-items-center flex-wrap">
-                <h6 class="mb-0"><%= "#{parse_wallet_number(wallet.wallet_number)}" %></h6>
```

Comment:
Why you deleted method parse_wallet_number? It parses our wallet number to view of 1111 - 2222 - 3333 - 4444.